Components/Core/Spells/Equivalencies.lua

```diff
@@ -196,30 +196,31 @@ TMW.BE = {
 		},
 		Slowed = {
 			   -116, -- Frostbolt                           (mage, frost)
-			   -120, -- Cone of Cold                        (mage, frost)
 			  -1715, -- Hamstring                           (warrior, arms)
 			   2120, -- Flamestrike                         (mage, fire)
-			  -3409, -- Crippling Poison                    (rogue, assassination)
-			  -3600, -- Earthbind                           (shaman, general)
-			  -5116, -- Concussive Shot                     (hunter, beast mastery/marksman)
+			   3409, -- Crippling Poison                    (rogue, assassination)
```

Comment:
Checking by name (prefix with `-`) is intended for the following abilities where there are lots of NPC abilities with the same name that have the same kind of effect:
- Crippling Poison
- Earthbind
- Concussive Shot
- Piercing Howl (10567 being an exception that I'm willing to live with being a false positive)
- Mind Flay (a few exceptions here, again willing to live with for the benefit of 95%+ of them being right
- Frost Shock